db/post_migrate/20210614050646_change_boolean_custom_fields_to_ints.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class ChangeBooleanCustomFieldsToInts < ActiveRecord::Migration[6.1]
+  def up
+    set_topic_first_approved_post_ids
+    set_topic_needs_approval_to_int
+  end
+
+  def set_topic_needs_approval_to_int
+    topic_needs_approval_custom_field_ids = execute("SELECT topic_id FROM topic_custom_fields WHERE name='category_expert_topic_post_needs_approval' AND value = 't'").values.flatten
```

Comment:
I know I run a bunch of queries here, but in reality it only loops over existing category expert custom fields and no site is using this plugin with _that many_ category expert posts to where it would cause a problem.

It's so much easier for me to write it this way with lots of queries than to try to do it all at once.